.setup/vagrant/db_inserts.sql

```diff
@@ -4,25 +4,25 @@ INSERT INTO sections_registration(sections_registration_id)
     VALUES (1);
 
 -- Add all instructors / TAs, administrators
-INSERT INTO users (user_id, user_firstname, user_lastname, user_email, user_group, registration_section, rotating_section)
-    VALUES ('developer', 'Developer', 'Jackson', 'instructor@email.edu', 0, NULL, NULL);
+INSERT INTO users (user_id, user_password, user_firstname, user_lastname, user_email, user_group, registration_section, rotating_section)
+    VALUES ('developer', '$2y$10$DuPIEHfBo5ka582lxrY5Au0roVsH3x7DAuoIVKHKrlM6wYu7zHGWG', 'Developer', 'Jackson', 'instructor@email.edu', 0, NULL, NULL);
```

Comment:
these are just used on Travis, but to distribute them we'd have to write a PHP script to generate and insert these after this db_inserts.sql. I'd prefer just write a comment that says "use password_hash('user_id', PASSWORD_DEFAULT) to generate passwords" at the top as these shouldn't ever change.